spec/classes/mod/worker_spec.rb

```diff
@@ -126,7 +126,8 @@
       it { is_expected.to contain_file('/etc/httpd/conf.modules.d/worker.conf').with(content: %r{^<IfModule mpm_worker_module>$}) }
       it { is_expected.to contain_file('/etc/httpd/conf.modules.d/worker.conf').with(content: %r{^\s+ServerLimit\s+10$}) }
       it { is_expected.to contain_file('/etc/httpd/conf.modules.d/worker.conf').with(content: %r{^\s+StartServers\s+11$}) }
-      it { is_expected.to contain_file('/etc/httpd/conf.modules.d/worker.conf').with(content: %r{^\s+MaxClients\s+12$}) }
+      it { is_expected.not_to contain_file('/etc/httpd/conf.modules.d/worker.conf').with(content: %r{^\s+MaxClients}) }
```

Comment:
It's better to expect a file to be there without certain content. The big reason is that your example will also pass if the file isn't present in the catalog at all.
```suggestion
      it { is_expected.to contain_file('/etc/httpd/conf.modules.d/worker.conf').without(content: %r{MaxClients}) }
```

On a side note: this whole file should be rewritten IMHO to have a single `it` statement with one long `content` block that has the full generated template, but that's not part of this PR.

Reply:
Ack done